.gitignore

```diff
@@ -12,3 +12,6 @@
 
 # Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
 .glide/
+
+# Idea IDE files
+.idea/
```

Comment:
I'm using pycharm with the Go plugin. It has everything I seem to need so far, but I'll try the others out too. I'm hesitating on Atom as it's too heavy for a simple editor and too light for an IDE from what I see (as well as because of the inevitable configuration phase that I will go through). I might give it a shot, though.